### ISSUE TITLE
Card Acquisition Transaction - bad request

### ISSUE THREAD

**Issue by Pret88** [Adyen/adyen-dotnet-api-library#82]
Hi, i just want use CardAcquisitionTransaction,I made a request according to the documentation https://docs.adyen.com/point-of-sale/build-your-integration/acquire-card-details/
example:
```
var request = new SaleToPOIRequest()
            {
                MessageHeader = new MessageHeader()
                {
                    ProtocolVersion = "3.0",
                    MessageType = "Request",
                    MessageClass = "Service",
                    MessageCategory = "CardAcquisition",
                    SaleID = "USER" + input.SaleId,
                    POIID = input.POIID,
                    ServiceID = "AC" + input.TerminalTransactionID.ToString()
                },
                MessagePayload = new CardAcquisitionRequest()
                {
                    SaleData = new SaleData()
                    {
                        SaleTransactionID = new TransactionIdentification()
                        {
                            TransactionID = "AC" + input.TerminalTransactionID.ToString(),
                            TimeStamp = DateTime.Now
                        }/*,
                        TokenRequested = "Customer"*/
                    },
                    CardAcquisitionTransaction = new CardAcquisitionTransaction()
                    {
                        
                    }

                },
                SecurityTrailer = new ContentInformationType() { }
            };
```
but response is with errors:
```
"MessagePayload": {
    "Response": {
      "AdditionalResponse": "tid=75102760&transactionType=GOODS_SERVICES&backendGiftcardIndicator=false&expiryYear=2028&posAmountGratuityValue=0&giftcardIndicator=false&paymentMethodVariant=maestro&expiryDate=2%2f2028&cardHolderName=N%2fA&applicationPreferredName=ms%20en&txtime=08%3a57%3a59&iso8601TxDate=2019-05-20T06%3a57%3a59.0000000%2b0000&cardType=maestro&posOriginalAmountValue=0&txdate=20-05-2019&paymentMethod=mc&merchantReference=AC77&applicationLabel=ms%20en&expiryMonth=02&cardSummary=9990&warnings=At%20SaleToPOIRequest.CardAcquisitionRequest.CardAcquisitionTransaction%2c%20field%20LoyaltyHandling%3a%20Field%20not%20supported%0aAt%20SaleToPOIRequest.CardAcquisitionRequest.CardAcquisitionTransaction%2c%20field%20ForceCustomerSelectionFlag%3a%20Field%20not%20supported%0aAt%20SaleToPOIRequest.CardAcquisitionRequest.CardAcquisitionTransaction%2c%20field%20CashBackFlag%3a%20Field%20not%20supported&posAuthAmountCurrency=EUR&posAmountCashbackValue=0&posEntryMode=ICC&cardScheme=maestro&cardBin=679999&posAuthAmountValue=0",
      "Result": "Success",
      "ErrorCondition": null
    },
```
TotalAmount is not supported and many others fields
Can you please fix this problem

**Comment by AlexandrosMor** [comment]
Hello @Pret88.
Thank you for reporting this issue. We will investigate it and come back to you

kind regards,
Alexandros Moraitis

**Comment by AlexandrosMor** [comment]
@Pret88 
We released the version 3.0 of Adyen library. Is it possible to test with the latest version? 
Thank you in advance,
Alexandros Moraitis
